packages/core/admin/server/controllers/api-token.js

```diff
@@ -100,6 +101,7 @@ module.exports = {
       attributes.description = trim(body.description);
     }
 
+    // TODO: can yup handle this and throw an error if an unexpected field is included?
```

Comment:
If you want to throw on unknown attributes, you can add .strict() after the .noUnknown in the api-token validators